README.md

```diff
@@ -102,12 +112,23 @@ spec:
 | `LEGO_KUBE_API_URL` | n | `http://127.0.0.1:8080` | API server URL |
 | `LEGO_LOG_LEVEL` | n | `info` | Set log level (`debug|info|warn|error`) |
 
-
 ## Full deployment examples
 
 - [Nginx Ingress Controller](examples/nginx/)
 - [GCE Load Balancers](examples/gce/)
 
+## Troubleshooting
+
+When interacting with *kube-lego*, its a good idea to run with `LEGO_LOG_LEVEL=debug` for more verbose details.
+Additionally, be aware of the automatically created resources (see environment variables) when cleaning up or testing.
+
+Possible resources for *help*:
+
+* `#coreos` on freenode
```

Comment:
You make a good point. The way I see it:

* Either we create a chat location for kube-lego
* We get explicit permission from other places to post their channel here.
* We rename as "k8s chat channels" instead of "places for help"
* We do absolutely nothing, which is utterly unacceptable. People should have a place to go for aid on working with kube-lego.

I included these places as a lot of people who have used kube-lego hang out in these channels and have been helpful to me and others.